pkg/parser/parser_test.go

```diff
@@ -538,6 +538,17 @@ func TestAdminStmt(t *testing.T) {
 		{"admin set bdr role secondary", true, "ADMIN SET BDR ROLE SECONDARY"},
 		{"admin unset bdr role", true, "ADMIN UNSET BDR ROLE"},
 		{"admin show bdr role", true, "ADMIN SHOW BDR ROLE"},
+		// for alter job
```

Comment:
Please add multi jobs alter test cases.

Reply:
Currently, we only support modify one job's config each time, will support multi jobs later if necessary.